src/plugins/data_view_field_editor/public/components/field_editor/form_fields/script_field.tsx

```diff
@@ -259,7 +259,7 @@ const ScriptFieldComponent = ({ existingConcreteFields, links, placeholder }: Pr
               }
               fullWidth
             >
-              <CodeEditor
+              <CodeEditor // TODO theme provided in src/plugins/data_view_field_editor/public/open_editor.tsx
```

Comment:
Works now. The TODO can be removed:

<img width="500" alt="Screenshot 2023-06-28 at 10 15 02" src="https://github.com/elastic/kibana/assets/1415710/246117bc-c58f-4f0a-a038-e00971316405">